SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularitySlaveUsageWithCalculatedScores.java

```diff
@@ -121,6 +129,10 @@ SingularitySlaveUsage getSlaveUsage() {
     return diskInUseScore;
   }
 
+  long getTimestamp() {
```

Comment:
you should be able to just do getSlaveUsage().getTimestamp() instead of having to store it in two places